Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml

```
@@ -139,6 +139,14 @@
   metamorphicMaxFillLevels: 5
   metamorphicFillBaseName: fill-
   metamorphicChangeColor: false
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
```

yamllint

[error] wrong indentation: expected 8 but found 6

(indentation)